pandas/tests/frame/test_constructors.py

```diff
@@ -713,6 +716,29 @@ def test_constructor_period(self):
         assert df["a"].dtype == a.dtype
         assert df["b"].dtype == b.dtype
 
+    @pytest.mark.parametrize(
+        "data,dtype",
+        [
+            (Period("2020-01"), PeriodDtype("M")),
+            (Interval(left=0, right=5), IntervalDtype("int64")),
+            (
+                Timestamp("2011-01-01", tz="US/Eastern"),
+                DatetimeTZDtype(tz="US/Eastern"),
+            ),
+        ],
+    )
+    def test_constructor_period_data(self, data, dtype):
```

Comment:
can you change the name -> test_constructor_scalar_data

Reply:
@justinessert here.

Reply:
done